src/commands/upload-graphql-shema.js

```diff
@@ -0,0 +1,71 @@
+const FaunaCommand = require('../lib/fauna-command.js')
+const { flags, Command } = require('@oclif/command')
+const fetch = require('node-fetch')
+const fs = require('fs')
+const path = require('path')
+const { errorOut } = require('../lib/misc.js')
+
+class UploadGraphQLSchemaCommand extends FaunaCommand {
+  allowedExt = ['.graphql', '.gql']
+
+  async run() {
+    try {
+      const { graphqlPath } = this.args
+      const { mode, graphqlHost } = this.flags
+
+      if (!this.allowedExt.includes(path.extname(graphqlPath))) {
+        errorOut(
+          'You must provide a Graphql file (`.graphql` or `.gql`) to upload a schema'
```

Comment:
```suggestion
          'Your GraphQL schema file must include the `.graphql` or `.gql` extension.'
```

Reply:
@faunaee i copied this text from vscode plugin https://github.com/fauna/vscode/blob/master/src/uploadGraphqlSchemaCommand.ts#L16
should it be consistent?

Reply:
I would say so, yes.